src/components/FormProduct/index.jsx

```diff
@@ -10,38 +14,40 @@ export const FormProduct = () => {
 
   const [product, setProduct] = useState(initialStateProduct);
 
-  const [error, setError] = useState(false);
-
+  const [error, setError] = useState('');
   const handleInputProduct = (e) => {
     setProduct({
       ...product,
       [e.target.name]: e.target.value,
     });
+    setError(
+      e.target.value ? (
+        ''
+      ) : (
+        <span aria-live="polite">All fields are required</span>
+      ),
+    ); //Required field. Immediate validation
```

Comment:
I don't really understand this comment 😅.

Maybe should it be before the `setError` call? Maybe it should be more explicit? No biggie though, but, still worth checking out.